spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRowLevelOperationsWithLineage.java

```diff
@@ -89,9 +129,6 @@ public static void setupSparkConf() {
   @BeforeEach
   public void beforeEach() {
     assumeThat(formatVersion).isGreaterThanOrEqualTo(3);
-    // ToDo: Remove these as row lineage inheritance gets implemented in the other readers
-    assumeThat(fileFormat).isEqualTo(FileFormat.PARQUET);
```

Comment:
Don't need these assumptions since the parameters are being explicitly overriden now